IdentityCore/src/telemetry/MSIDTelemetry.m

```diff
@@ -182,9 +182,10 @@ - (void)dispatchEventNow:(NSString *)requestId
         {
             for (NSString *propertyName in [event.propertyMap allKeys])
             {
-                BOOL isPii = [MSIDTelemetryPiiRules isPii:propertyName];
+                BOOL isPii = [MSIDTelemetryPiiOiiRules isPii:propertyName];
```

Comment:
nit: perhaps it makes sense to also add a method to MSIDTelemetryPiiOiiRules that checks for both PII and OII for cases when we check for both and handle them in the same way.

Reply:
Sure, let me do it :)